public/static/locales/isl/parallel.json

```diff
@@ -0,0 +1,7 @@
+{
+    "sign-in-hello": "Þú getur fylgst með framvindu þinni með því að búa til aðgang.",
+    "sign-in-hello-link": "Smelltu hér til að skrá þig inn eða búa til aðgang.",
+    "google-form-link": "Smelltu hér til að fylla út GoogleForms-skjalið.",
```

Comment:
"GoogleForms" -> "Google Forms"

Reply:
Changed in 701afa8f2c3971aaca11d7a957262b124fd5f9b2